index.html

```diff
@@ -140,50 +140,91 @@ <h2>Why Git?</h2>
                     </p>
                 </section>
             </section>
-            <section>
+            <section id="7">
+                <h2>Demo</h2>
+                <p>
+                <ul>
+                    <li class="fragment">Config the git</li>
+                    <li class="fragment">Forking Repositories on Github</li>
+                    <li class="fragment">Cloning Them</li>
+                    <li class="fragment">Making Changes</li>
+                    <li class="fragment">Commiting those Changes</li>
+                    <li class="fragment">Adding Remotes</li>
+                    <li class="fragment">Checkout</li>
+                    <li class="fragment">Pushing Your Changes</li>
+                    <li class="fragment">Branching</li>
+                    <li class="fragment">Making Pull Requests</li>
+                    <li class="fragment">Pulling New Changes and more ..</li>
+                </ul>
+                </p>
+            </section>
+            <section id="8">
                 <h2>Git Workflow</h2>
                 <section>
-                <span>Obtain Repo
-                  <pre><code class="hljs" data-trim>git init</code></pre>
-                  <pre><code class="hljs" data-trim>git clone</code></pre>
-                </span>
+                <span>Configuring git
+                  <pre><code class="hljs" data-trim>git config --global user.name "John Doe"</code></pre>
+                  <pre><code class="hljs" data-trim> git config --global user.email "johndoe@example.com"</code></pre>
+                  <pre><code class="hljs" data-trim> git config --global core.editor "nano"</code></pre>
+                  <p>
+                  <ul>
+                      <li class="fragment"> This is done for git to identify who is making the commits, for people who will check later on. We recommend to go with nano in the beginning for the editor of choice.</li>
```

Comment:
I think you might want to say:

`This is done for git to identify who is making the commits. For newbies, we recommend going with nano at the beginning as the editor of choice.`

Reply:
Yours sounds better. Implementing